src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -81,7 +81,7 @@ object RpmPlugin extends AutoPlugin {
     rpmConflicts := Seq.empty,
     rpmSetarch := None,
     rpmChangelogFile := None,
-    rpmBrpJavaRepackJars := false,
+    rpmBrpJavaRepackJars := true,
```

Comment:
@muuki88 Right, I negated the default because this is basically the behavior people had without actually knowing it so far. The time impact is pretty major if you're doing daily builds. I was waiting for one of you guys to set the tone on this, i'll revert the default to `false`.